### ISSUE TITLE
can't get it work with .authinfo 

### ISSUE THREAD

**Issue by MarinosK** [dengste/org-caldav#229]
Not sure if that's a bug or a wrong configuration of mine - I never had to do this manually, other programs ask me if I want keep my credentials to authinfo and this is handled automatically. Anyhow: 

I have this in my .authinfo

`machine nextcloud.MY_SERVER/remote.php/dav/calendars/USERNAME:443 port https login USERNAME password PASSWORD`

but org-caldav still asks username and password - doesn't look at this file 

Note the .authinfo is the standard used for encyrption and is populated wth similar entires and starred out passwords. So I also tried with a seperate .authinfo.gpg only having the above entry, and added to my auth-sources like this 

```
(setq auth-sources
      '((:source "~/.authinfo" "~/.authinfo.gpg")))
```

**Comment by pmeckoni** [comment]
Shouldn't the authinfo file only have the machine name and not the entire url?

```
machine MYnextcloudSERVER.net:443 port https login USERNAME password PASSWORD
```

**Comment by lytex** [comment]
Yes, I got mine working with `org-caldav-url` set to  `https://nextcloud.MY_SERVER/remote.php/dav/calendars/USERNAME` and then in .authinfo.gpg / .authinfo I have `nextcloud.MY_SERVER:443`

It's a little confusing, `org-caldav-url` has `https://`, full path `remote.php/dav/calendars/USERNAME`, and in your .authinfo you only have to specify server and port without any path
